helm-charts/konk/values.yaml

```diff
@@ -25,6 +25,11 @@ apiserver:
     # runAsNonRoot: true
     # runAsUser: 1000
   startupProbe: true
+  remoteHeaders:
+    - --requestheader-extra-headers-prefix=X-Remote-Extra-
+    - --requestheader-group-headers=X-Remote-Group
+    - --requestheader-username-headers=X-Remote-User
+  extraFlags: []
```

Comment:
If this were a map it could be additive, which might be easier to deal with in a system of layered values overrides.

Reply:
I agree